runtime/vm/rasdump.c

```diff
@@ -628,9 +628,10 @@ populateRASNetData(J9JavaVM *javaVM, J9RAS *rasStruct)
 		j9sock_freeaddrinfo( &addrinfo );
 	}
 
-	endTime = j9time_current_time_millis();
-	if (endTime - startTime > RAS_NETWORK_WARNING_TIME) {
-		j9nls_printf(PORTLIB, J9NLS_INFO, J9NLS_VM_RAS_SLOW_NEWORK_RESPONSE, (int)(endTime - startTime)/1000);
+	endTime = j9time_hires_clock();
+	timeDiff = j9time_hires_delta(startTime, endTime, OMRPORT_TIME_DELTA_IN_MILLISECONDS);
```

Comment:
I suggest this should use `omrtime_hires_clock()` and `omrtime_hires_delta()` since this uses `OMRPORT_TIME_DELTA_IN_MILLISECONDS`.

Reply:
Fixed as suggested.
I also changed the commit title and the commit message.